examples/ELZ-ReferralServiceRequest-02-Patient-01.xml

```diff
@@ -0,0 +1,186 @@
+                <valueCodeableConcept>
+                    <coding>
+                        <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
+                        <code value="MC"/>
+                        <display value="Mobile Phone"/>
+                    </coding>
+                </valueCodeableConcept>
+            </extension>
+        </system>
+        <value value="0612324322"/>
+        <use value="home"/>
+    </telecom>
+    <telecom>
+        <system value="email"/>
+        <value value="XXX_Quigley@test.nl"/>
+        <use value="home"/>
+    </telecom>
+    <gender value="male"/>
+    <birthDate value="1985-11-04"/>
+    <deceasedBoolean value="false"/>
+    <address>
+        <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-AddressInformation.AddressType">
+            <valueCodeableConcept>
+                <coding>
+                    <system value="http://terminology.hl7.org/CodeSystem/v3-AddressUse"/>
+                    <code value="HP"/>
+                    <display value="Primary Home"/>
+                </coding>
+            </valueCodeableConcept>
+        </extension>
+        <use value="home"/>
+        <type value="both"/>
+        <text value="Knolweg 1003, 9999ZZ, STITSWERD"/>
+        <line value="Knolweg 1003">
+            <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-streetName">
+                <valueString value="Knolweg"/>
+            </extension>
+            <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-ADXP-houseNumber">
+                <valueString value="1003"/>
+            </extension>
+        </line>
+        <city value="STITSWERD"/>
+        <district value="STITSWERD"/>
+        <postalCode value="9999ZZ"/>
+        <country value="Nederland">
+            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification">
+                <valueCodeableConcept>
+                    <coding>
+                        <system value="urn:iso:std:iso:3166"/>
+                        <code value="NL"/>
+                        <display value="Nederland"/>
+                    </coding>
+                </valueCodeableConcept>
+            </extension>
+        </country>
+    </address>
+    <maritalStatus>
+        <coding>
+            <system value="http://terminology.hl7.org/CodeSystem/v3-MaritalStatus"/>
+            <code value="M"/>
+            <display value="Gehuwd"/>
+        </coding>
+    </maritalStatus>
+    <multipleBirthBoolean value="true"/>
+    <contact>
+        <relationship>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v3-RoleCode"/>
+                <code value="DOMPART"/>
+                <display value="Partner"/>
+            </coding>
+        </relationship>
+        <name>
+            <use value="official"/>
+            <text value=" Marieke van XXX_Quigley - Streng"/>
+            <family value="van Gestel">
```

Comment:
Shouldn't the extensions be specified in the same order as in the profile itself (in xml)?

Reply:
I've changed to order to be sure.